C/c4CAPI.cc

```diff
@@ -492,6 +492,18 @@ bool c4db_findDocAncestors(C4Database* database, unsigned numDocs, unsigned maxA
     });
 }
 
+// only used by tests
```

Comment:
Perhaps more accurate is "declared in c4Private.h and meant to be used by tests."